test/e2e/tests/grpc_test.go

```diff
@@ -132,3 +135,60 @@ func TestGRPC_Block_GetLatestHeight(t *testing.T) {
 		}
 	}
 }
+
+func TestGRPC_GetBlockResults(t *testing.T) {
+	testNode(t, func(t *testing.T, node e2e.Node) {
+		if node.Mode != e2e.ModeFull && node.Mode != e2e.ModeValidator {
+			return
+		}
+
+		client, err := node.Client()
+		require.NoError(t, err)
+		status, err := client.Status(ctx)
+		require.NoError(t, err)
+
+		first := status.SyncInfo.EarliestBlockHeight
+		last := status.SyncInfo.LatestBlockHeight
+		if node.RetainBlocks > 0 {
+			first++
+		}
+
+		ctx, ctxCancel := context.WithTimeout(context.Background(), time.Minute)
+		defer ctxCancel()
+		gRPCClient, err := node.GRPCClient(ctx)
+		require.NoError(t, err)
+
+		// GetLatestBlockResults
+		latestBlockResults, err := gRPCClient.GetLatestBlockResults(ctx)
+
+		require.GreaterOrEqual(t, last, latestBlockResults.Height)
+		require.NoError(t, err, "Unexpected error for GetLatestBlockResults")
+		require.NotNil(t, latestBlockResults)
+
+		successCases := []struct {
+			expectedHeight int64
+			requestHeight  int64
+		}{
+			{first, first},
+			{latestBlockResults.Height, latestBlockResults.Height},
```

Comment:
The expected and requested should always be the same, right? If so I suspect we only need `requestHeight` in the test cases.

Reply:
Yes fair, this is leftover from before but it's not needed now